traits/trait_set_object.py

```diff
@@ -38,31 +37,48 @@ def __init__(self, removed=None, added=None):
 
         if removed is None:
             removed = set()
+        self.removed = removed
+
         if added is None:
             added = set()
-
-        self.removed = removed
         self.added = added
 
 
+# Default item validator for TraitList.
+
+def accept_anything(item):
+    """
+    Item validator which accepts any item and returns it unaltered.
+    """
+    return item
+
+
 class TraitSet(set):
     """ A subclass of set that validates and notifies listeners of changes.
 
     Parameters
     ----------
-    value : set
-        The value for the set
-    validator : callable
-        Called to validate items in the set
+    value : iterable
+        Iterable providing the items for the set.
+    item_validator : callable, optional
+        Called to validate and/or transform items added to the list. The
+        callable should accept a single item from the list and return
+        the transformed item, raising TraitError for invalid items. If
+        not given, no item validation is performed.
     notifiers : list of callable
         A list of callables with the signature::
 
             notifier(removed, added)
 
+        If this argument is not given, the list of notifiers is initially
+        empty.
```

Comment:
I see you have removed the notes on what `removed` and `added` are in individual methods.
Are you going to mention what `removed` and `added` are here or is the description added somewhere?

Reply:
I just added a description here. Should be enough to understand what they are.